examples/freesurfer_segs/freesurfer_provenance.jsonld

```diff
@@ -0,0 +1,53 @@
+{
+	"@context": [
+		"https://purl.org/nidash/bidsprov/context.json",
+		{
+			"freesurfer": "https://surfer.nmr.mgh.harvard.edu/",
+			"description": "http://purl.org/dc/terms/description"
+		}
+	],
+	"BIDSProvVersion": "1.0.0",
+	"records": {
+		"prov:Agent": [
+			{
+				"@id": "RRID:SCR_001847",
+				"@type": "prov:SoftwareAgent",
+				"label": "Freesurfer",
+				"version": "6.0.0"
+			}
+		],
+		"prov:Activity": [
+			{
+				"@id": "niiri:ff2754fe-1585-11eb-b1e7-1094bbf2086c",
+				"description": "Freesurfer segmentation statistics",
+				"startedAtTime": "2018-11-27T21:42:49",
+				"wasAttributedTo": "RRID:SCR_001847",
```

Comment:
we used `wasAssociatedWith` to record the fact that an activity is a running instance of an Agent

an example [here](https://github.com/bids-standard/BEP028_BIDSprov/blob/ebd8b856caa5999583fd3319c6e7973cbe5a2514/examples/spm_default/copies.json#L27)

And the def of this term in our common context [here](https://github.com/bids-standard/BEP028_BIDSprov/blob/ebd8b856caa5999583fd3319c6e7973cbe5a2514/context.json#L27)